stan/math/opencl/kernel_generator/is_valid_expression.hpp

```diff
@@ -2,7 +2,7 @@
 #define STAN_MATH_OPENCL_IS_USABLE_AS_OPERATION
```

Comment:
Not added in this PR but the header of this file doesnt match the filename.

Reply:
Fixed this and some other header guards.